compile.sh

```diff
@@ -1,20 +1,23 @@
 #!/bin/sh -x -e
 
-# Note that the build system needs to be Apple Silicon with Rosetta 2 enabled.
+# Note that the build system needs to be BigSur or newer, Intel or Apple Silicon
 #
 # Before building, be sure the checkout sources and configure the system:
 #
 # 1) Install the latest version of Xcode.
 #
 # 2) Install autoconf, automake, pkgconfig, and glibtool to ${BUILD_TOOLS_PREFIX} (eg: through MacPorts)
 #    Eg with MacPorts:
-#        base $ ./configure --prefix=/opt/buildX11 && make -j$(sysctl -n hw.activecpu) && sudo make install
-#        $ sudo nano -w /opt/buildX11/etc/macports/macports.conf # Edit applications_dir        /Applications/MacPorts/BuildX11
+#        $ cd /tmp && rm -rf /tmp/macports-base && git clone -b release-2.6 https://github.com/macports/macports-base.git && cd macports-base
```

Comment:
I suggest just nuking this section completely and replacing it with a pointer to your new script in your other pull request.